.github/workflows/build.yml

```diff
@@ -79,28 +117,83 @@ jobs:
           username: ${{ env.REGISTRY_USER }}
           password: ${{ env.REGISTRY_PASSWORD }}
 
+      - name: Echo outputs
+        if: github.event_name != 'pull_request'
+        run: |
+          echo "${{ toJSON(steps.push.outputs) }}"
+  combine-manifests:
+    name: Combine manifests
+    runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request'
+    needs:
+      - calculate-tags
+      - push-ghcr
+    steps:
+      - uses: sigstore/cosign-installer@v3.7.0
+
+      - name: Install skopeo & podman
+        run: |
+          sudo apt update
+          sudo apt install -y skopeo podman
+
+      # Workaround bug where capital letters in your GitHub username make it impossible to push to GHCR.
+      # https://github.com/macbre/push-to-ghcr/issues/12
+      - name: Lowercase Registry
+        id: registry_case
+        uses: ASzc/change-string-case-action@v6
+        with:
+          string: ${{ env.IMAGE_REGISTRY }}
+
+      # Login with docker for cosign
       - name: Login to GitHub Container Registry
         uses: docker/login-action@v3
-        if: github.event_name != 'pull_request'
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      # Sign container
-      - uses: sigstore/cosign-installer@v3.7.0
-        if: github.event_name != 'pull_request'
+      # Login with podman for buildah
+      - name: Log in to ghcr.io
+        uses: redhat-actions/podman-login@4934294ad0449894bcd1e9f191899d7292469603 # v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Combine manifests
```

Comment:
this step takes the tags with the arch appended and creates a new manifest file with both arch's, with a new tag that points to both arch versions. Client picks the appropriate one automatically